pandas/tests/indexing/test_loc.py

```diff
@@ -778,3 +778,9 @@ def test_loc_setitem_empty_append_raises(self):
         msg = "cannot copy sequence with size 2 to array axis with dimension 0"
         with pytest.raises(ValueError, match=msg):
             df.loc[0:2, 'x'] = data
+
+    def test_loc_getitem_interval_index(self):
+        """ GH25087, test get_loc returns key error for interval indexes"""
+        idx = pd.interval_range(0, 1.0)
+        with pytest.raises(KeyError):
+            idx.get_loc('gg')
```

Comment:
Instead of testing here, can you add this as a test case to `test_get_loc_value` in `pandas/tests/indexes/interval/test_interval.py`:

https://github.com/pandas-dev/pandas/blob/a09a07e4dabe91a8cfe46bec40414a2ddc4225a3/pandas/tests/indexes/interval/test_interval.py#L416

Reply:
I've just implemented this in 93f75ea